lib/cloud_controller/config_schemas/api_schema.rb

```diff
@@ -257,8 +257,6 @@ class ApiSchema < VCAP::Config
             enabled: bool,
             optional(:public_endpoint) => enum(String, NilClass),
             optional(:private_endpoint) => enum(String, NilClass),
-            optional(:username) => enum(String, NilClass),
```

Comment:
We're wondering why you deleted these two from here and why we don't see the two new properties that are being added in https://github.com/cloudfoundry/capi-release/pull/129?

Noticed that you switched to accessing these as a hash so it's probably not strictly necessary, but leads to errors like this if we try to access it via the config object interface:

```
[7] pry(VCAP::CloudController)> Config.config.get(:bits_service, :signing_key_secret)
VCAP::CloudController::Config::InvalidConfigPath: "bits_service.signing_key_secret" is not a valid config key
```

Reply:
Hm. All good questions. @eric-cartman, I think you've worked on this mainly with @idev4u . Could you comment or update the PR accordingly? I won't be able to work in this while I'm on leave. Thanks. /Cc @smoser-ibm

Reply:
@eric-cartman @smoser-ibm 

Any updates?

Reply:
@tcdowney / @petergtz 
Currently looking into this.